src/Microsoft.TemplateSearch.Common/Abstractions/TemplatePackageSearchData.cs

```diff
@@ -11,6 +11,7 @@ namespace Microsoft.TemplateSearch.Common
     /// <summary>
     /// Template package searchable data.
     /// </summary>
+    [System.Diagnostics.DebuggerDisplay("{Name}@{Version}")]
```

Comment:
This workflow deals with large lists of these items, having a debugger visualization is very handy.